debug/py314_debug.py

```
@@ -0,0 +1,23 @@
+}
+
+for name, src in cases.items():
+    with tempfile.NamedTemporaryFile(suffix=".py", mode="w", delete=False) as f:
+        f.write(src); sp = f.name
+    pp = sp + "c"
+    py_compile.compile(sp, cfile=pp, doraise=True)
+    with open(pp, "rb") as f: data = f.read()
+    code = marshal.load(io.BytesIO(data[16:]))
+    print(f"=== {name} ===")
+    for i in dis.get_instructions(code):
+        print(f"  {i.offset:3d}  {'>>':2s}  {i.opname:<30} arg={str(i.arg):6} argval={str(i.argval):<20} is_jump_target={i.is_jump_target}")
```

Ruff (0.15.6)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)